static/app/components/search/sources/formSource.spec.tsx

```diff
@@ -35,7 +37,7 @@ describe('FormSource', function () {
   });
 
   afterEach(function () {
-    ActionCreators.loadSearchMap.mockRestore();
+    (ActionCreators.loadSearchMap as jest.Mock).mockRestore();
```

Comment:
I've just instead been switching this to `Jest.restoreAllMocks()`

Reply:
Ah nice, we should do that indeed. I'll make a note for myself to change this next time, the casts are 😞